administrator/components/com_joomlaupdate/extract.php

```diff
@@ -0,0 +1,2046 @@
+<?php
+/**
+ * @package         Joomla.Administrator
+ * @subpackage      com_joomlaupdate
+ *
+ * @copyright       (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Uncomment to enable debugging and prevent extract.php from being overwritten during the update
+//define('_JOOMLA_UPDATE_DEBUG', 1);
```

Comment:
```suggestion
// define('_JOOMLA_UPDATE_DEBUG', 1);
```
Unfortunately the PHPCS check in drone doesn't like that, even if it's right like you did.

Reply:
Maybe if you fix that with a space then the next complaint by PHPCS will be that it's a multiline comment so "/* */" should be used.

I don't know if we can switch off checks by certain comments like we can for the SCSS linter.

Reply:
Ah, well, let's make the poor, sad robot happy again 😛